hivemind/p2p/p2p_daemon_bindings/keys.py

```diff
@@ -0,0 +1,91 @@
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from enum import Enum, unique
+
+from hivemind.proto import crypto_pb2 as protobuf
+
+
+@unique
+class KeyType(Enum):
+    RSA = 0
+    Ed25519 = 1
+    Secp256k1 = 2
+    ECDSA = 3
+    ECC_P256 = 4
+
+
+class Key(ABC):
```

Comment:
Ideally, it would be great to replace these classes with the interfaces of the `cryptography` module (since we already use it for RSA signatures of DHT records).

However, this may require a lot of work, so I'm okay if you won't do that in this PR.